src/Renderer/MaterialLayer.js

```diff
@@ -1,62 +1,33 @@
 import * as THREE from 'three';
 import { ELEVATION_MODES } from 'Renderer/LayeredMaterial';
-import { checkNodeElevationTextureValidity, insertSignificantValuesFromParent } from 'Parser/XbilParser';
+import { checkNodeElevationTextureValidity, insertSignificantValuesFromParent, computeMinMaxElevation } from 'Parser/XbilParser';
 import CRS from 'Core/Geographic/Crs';
 
 export const EMPTY_TEXTURE_ZOOM = -1;
 
 const pitch = new THREE.Vector4();
 
-function defineLayerProperty(layer, property, initValue, defaultValue) {
-    let _value = initValue !== undefined ? initValue : defaultValue;
-    Object.defineProperty(layer, property, {
-        get: () => _value,
-        set: (value) => {
-            if (_value !== value) {
-                _value = value;
-            }
-        },
-    });
-}
 
-class MaterialLayer {
```

Comment:
`MaterialLayer` is still called in some tests :

- _test/unit/demutils.js_ at line 62
- _test/unit/layeredmaterialnodeprocessing.js_ at line 54
- _test/unit/layerupdatestrategy.js_ at line 57

Reply:
yes, thanks